.github/workflows/builder.yml

```diff
@@ -48,18 +48,6 @@ jobs:
         with:
           ignore-dev: true
 
-      - name: Generate meta info
-        shell: bash
-        run: |
-          echo "${{ github.sha }};${{ github.ref }};${{ github.event_name }};${{ github.actor }}" > OFFICIAL_IMAGE
-
-      - name: Signing meta info file
-        uses: home-assistant/actions/helpers/codenotary@master
-        with:
-          source: file://${{ github.workspace }}/OFFICIAL_IMAGE
-          asset: OFFICIAL_IMAGE-${{ steps.version.outputs.version }}
-          token: ${{ secrets.CAS_TOKEN }}
```

Comment:
We still have the file at the image, but since we never check this file, make also not a lot sense to port that into cosign

Reply:
We actually do use this, it is used in system info to show the installation type (which will show "Unsupported Third Party Container" in that case).